Create actions.md (#1)
cambiatus/cambiatus.github.io: features/actions.md

# Claims

Claims are the main way to create new tokens for your community. We use the concept of Objectives and Actions.

### Objectives

Objectives are what you want to achieve with your actions. All actions must be done towards a bigger objective. We recommend giving a through description to your objectives and try to add an expected result for it: a **good** result and a **great** result. This will help you get a better grasp on how your expectations are aligned with the reality.

### Actions

Actions are the definition of exactly what the community must do to reach those objectives. We made actions in a way that they are very flexible and allow for different use cases. For that we added a few options:

**_Validity_ of an action. This can be by _date_ and _quantity_**

>   - _Date_: its pretty straightforward. The action can be claimed until a certain date. After that date is met, its no longer possible to open new claims. An important part is that the claims themselves can be validated after this date is met. Only new claims that cannot be created.

Reviewer: Here is my suggestion for this section: 
- _Date_: An action can be claimed until certain date is reached. After the set date, claims cannot be opened. The validation of opened claims can occur after the set date. The opening of new claims cannot occur after the set date.

Author: What you think of removing the repetition?

```suggestion
  - _Date_: An action can be claimed until certain date is reached. After the set date, claims cannot be opened. The validation of opened claims can occur after the set date.
```

Reviewer: Good idea!

>   - _Quantity_ : Maybe we should have called this _usages_, but it determines the number of times this action can be **Approved or Disapproved**. That means that the users can claim all they want, but only the specified number can lead to rewards. So lets say we configure this action to have `Quantity of 3`; we can have 100s of claims opened, but after the 3 one is voted fully (approved or disapproved) the rest will be "unvoteable:, as the action will change its status to "completed" (that is set the flag `is_completed` to `true`.

Reviewer: @lucca65 assuming this documentation is for onboarding purposes (partners and team). 

Here is my suggestion for this section:
_Quantity_ :  Determines the number of times an action can be **Approved or Disapproved**. Users can only claim an action and be rewarded the amount of times specified in quantity. An example is: an action that have `Quantity of 3` votes; This action can have 100s of opened claims, but after the first 3 votes (approved or disapproved) all of the opened claims, not voted yet, will become "unvoteable", and the action status will change to "completed".

Question:
- Claims that have "n" votes, independently of the votes direction (approved or disapproved), if "quantity = n" the status of the action changes to "completed", correct?

Author: That's what I was trying to clarify on that part:

> That means that the users can claim all they want, but only the specified number can lead to rewards.

About your question:

>  n" the status of the action changes to "completed", correct?

So its not about **votes**. Its about **claims**. 

Lets say you create an action called "Plant a tree", that belongs to an objective called "Plant 2 trees". In this case, we configure that action to have `quantity` as 2. That is, after the community has verified that exactly 2 trees were planted, the action will be considered completed. 

Lets say that this community had high engagement, and a lot of users have claimed to plant trees. Lets assume that we had 10 claims open. 

Now the verifiers of that **action** will have check and give their vote on each **claim**. Lets say that they disapprove the first, approved the second. When its time to vote for the third one, their vote will fail. The action be considered completed.

All those claims will be unvotable

Author: I've updated this section @victorgdev let me know what you think

  
**Verification**

> Verification is the process on which the community decides if the action was done or not. The verification can be *Automatic* if its going to be validated automatically by some software, or it can be *Manual*, which requires human validation.

Reviewer: Here is my suggestion for this section:
Verification is the process by which the community decides if an action is completed or not. The verification process can be *Automatic* if validated by software, or *Manual*, which requires human validation.

Author: Ok!
